### DISCUSSION TITLE
Help customizing borders

### DISCUSSION THREAD

**Discussion by Tesco-epic** [i3/i3#6206] [Q&A]
Hello all, I'm just wondering if rounded corners are a thing? I read online that I3-Gaps adds rounded borders, installed it and added border_radius 2 to my config, but no matter what i try from online it will not do it.
Then i heard that something called picom can do it, so installed that, but when it trys to run it just says "another composite manager is already running", i know i3 is a desktop environment but i don't know what a composite manager is.

If anyone could help me figure out how to get gaps working or set up picom to work that would be greatly appreciated!

![1723755104](https://github.com/user-attachments/assets/f9ab76d7-eb3a-4642-abe7-16fb2429222b)

**Comment by shimodii** [comment]
Hi, actually i3 is not a DE, it's a WM and doesn't come with any compositor, it's related to your distro or other packages you installed already I guess. And about gaps and round corners I don't have any info about round corners but for gaps you can use:
gaps inner <value>
gaps outer <value>

You can customize border color too, for this part you can check my dot files too.
